Extensions/TileMap/tilemapruntimeobject.ts

```diff
@@ -413,6 +413,7 @@ namespace gdjs {
     setOpacity(opacity: float): void {
       this._opacity = opacity;
       this._renderer.updateOpacity();
+      // TODO: Set all tile layers alpha and re-render tilemap.
```

Comment:
They ar not the same thing:
1. Setting the opacity of every layers
2. Setting the opacity of the map

"1." would blend layers into each other which would give a strange result.

Reply:
I added this TODO because today, setting the opacity of the tilemap object does not work.

Reply:
I couldn't find another place where alpha is useful: https://api.pixijs.io/@pixi/tilemap/CompositeTilemap.html#tile

Reply:
A best effort solution is fine, it was more about the todo comment that is missleading.